package/MDAnalysis/coordinates/PDB.py

```diff
@@ -397,6 +409,9 @@ def _read_frame(self, frame):
 
         pos = 0
         occupancy = np.ones(self.n_atoms)
+        tempfactor = np.empty(self.n_atoms)
+        tempfactor.fill(np.nan)  # Default value for unseen entry is a NaN
```

Comment:
Ideally I would want to do the same thing for occupancy, but I guess that's a regression? (unless we think it's a bug), how do we want to handle this?

Reply:
For example, [PLUMED RMSD](https://www.plumed.org/doc-v2.8/user-doc/html/_r_m_s_d.html) uses occupancy = 1 to select atoms included in the calculation and used occupancy = 0 to exclude. This may be a niche application. However, semantically it makes more sense to me to say "if we see an atom in a file and we don't have any other specific information, let's assume it exists (as opposed that it doesn't)". Thus I'd be in favor of **defaulting occupancy to 1**.

Reply:
If defaulting occupancy everywhere to 1 is a breaking change then I suppose it's better to make `ts.data['occupancy']` consistent with the current "default 0" and raise an issue to change it for 3.0.

Reply:
I'll be honest I'm kinda mindlessly trying to get 2.5.0 out - what's our easiest solution here? Set both of the ts.data default values to whatever gets populated in the topology attrs? I'd be willing to say that's a bug correction.

Reply:
Follow the topology as default is a safe choice and I would not find it controversial. At least it follows the principle of "minimum surprise".

Then we can decide afterwards if the current defaults make sense.